docs/getting-started/raster/services/export_api.md

```diff
@@ -4,7 +4,49 @@
  > :no_entry: **Authentication must be integrated in order to communicate with export service.**<br/>
 > **See the principles [here](/getting-started/raster/raster_authentication.md)**
 
-Export OpenAPI (Swagger)
+Export Service consists of /create POST message that allows users that have a valid token to create & download a GPKG file
```

Comment:
Why do we describe the open API that way? isn't it better to have a link for it?

Reply:
fixed